cms/templates/widgets/header.html

```diff
@@ -192,7 +192,7 @@ <h3 class="title"><span class="label"><span class="label-prefix sr">${_("Current
       </nav>
 
       % else:
-      <nav class="nav-not-signedin nav-pitch">
+      <nav class="nav-not-signedin nav-pitch" aria-label="${_('Account')}">
         <h2 class="sr">${_("You're not currently signed in")}</h2>
```

Comment:
It's inappropriate to have the value of this Heading indicate Logged in status.  I would change it to the value above "Account Navigation".  Navigation blocks such as these should be consistent and predictable.